src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
Although the end result looks correct, I don't understand why these changes are showing up as commits to this branch. This change should already been in the hexagon branch. Note that https://github.com/halide/Halide/compare/hexagon shows no changes to Argument.h relative to master. I'm really nervous there's something wrong with the branch. 

@zvookin @abadams @steven-johnson do an of you have a better sense of what is going on here? I'm not fully confident in my understanding of git.